routers/comments.py

```
@@ -0,0 +1,24 @@
+from fastapi import APIRouter, status
+from schemas.comment import CommentCreate, CommentUpdate
+
+router = APIRouter(prefix="/comments", tags=["Comments"])
+
+@router.get("/post/{post_id}") # 특정 게시글의 댓글 목록
+def get_comments(post_id: str, page: int = 1):
+    return {"message": f"{post_id}번 게시글의 {page}페이지 댓글 목록"}
+
+@router.post("/post/{post_id}", status_code=status.HTTP_201_CREATED)
+def create_comment(post_id: str, data: CommentCreate):
```

Ruff (0.14.11)

Unused function argument: `data`

(ARG001)